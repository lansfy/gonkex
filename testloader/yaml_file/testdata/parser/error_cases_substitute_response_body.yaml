- name: loader MUST substitute responseArgs to response components
  method: GET
  path: /some/path
  response:
    200: '{"hello": {{ .foo } }'
  cases:
    - responseArgs:
        200:
          foo: "value1"
