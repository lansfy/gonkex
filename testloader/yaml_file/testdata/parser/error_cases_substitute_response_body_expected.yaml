- Error: "process 'testdata/parser/error_cases_substitute_response_body.yaml': test 'loader MUST substitute responseArgs to response components #1': template: $.response.200:1: unexpected \"}\" in operand"
